packages/expo-router/src/link/__tests__/Link.test.web.tsx

```diff
@@ -1,3 +1,4 @@
+/** @jest-environment jsdom */
```

Comment:
This test uses `import { render } from '@testing-library/react';` which assumes there is a DOM environment.

It looks like there is progress being made for better support for SSR and RSC https://github.com/testing-library/react-testing-library/issues/1209